CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ You can find its changes [documented below](#060---2020-06-01).
 ### Maintenance
 
 - Standardized web targeting terminology. ([#1013] by [@xStrom])
+- Ported the X11 backend to `x11rb`. ([#1025] by [@jneem])
```

Comment:
Let's continue the tradition from the previous release and prefix backend specific changes.
Something like:
```md
- X11: Ported the backend from `xcb` to `x11rb`. ([#1025] by [@jneem])
```